mkosi/manifest.py

```diff
@@ -197,6 +214,21 @@ def record_deb_packages(self, root: Path) -> None:
 
             source_package.add(package)
 
+    def record_pkg_packages(self, root: Path) -> None:
+        package_descriptions = [str(p) for p in (root / "var/lib/pacman/local").glob("*/desc")]
+        packages = sorted(package_descriptions)
```

Comment:
```suggestion
        packages = sorted(root.joinpath("var/lib/pacman/local").glob("*/desc"))
```
should get rid of the mypy error (`package` is a str instead of the expected `Path`)

Reply:
Clearly I do need to install mypy so I don't need to rely on the CI :/

Reply:
It takes some getting used to, but as soon as a Python codebase gets a sufficient size it *does* help to keep ones sanity. :)